docs/standard/base-types/how-to-verify-that-strings-are-in-valid-email-format.md

```diff
@@ -58,7 +58,7 @@ Pattern `|(([0-9a-z]`: If the first character is not a quotation mark, match any
 
 Pattern `(\.(?!\.))`: If the next character is a period, match it. If it is not a period, look ahead to the next character and continue the match. `(?!\.)` is a zero-width negative lookahead assertion that prevents two consecutive periods from appearing in the local part of an email address.
 
-Pattern ``|[-!#\$%&'\*\+/=\?\^`\{\}\|~\w]``: If the next character is not a period, match any word character or one of the following characters: -!#$%&'*+/=?^`{}|~
+Pattern ``|[-!#\$%&'\*\+/=\?\^`\{\}\|~\w]``: If the next character is not a period, match any word character or one of the following characters: -!#$%&'\*+/=?^\`{}|~
```

Comment:
This edit should NOT produce any difference in the live docs. I just escaped them because GitHub editor was marking the list of the line as italic and code. These added backslashes must not appear in live docs. Can you confirm, @mairaw ?

Reply:
I think it looks okay:

If the next character is not a period, match any word character or one of the following characters: -!#$%&'*+/=?^`{}|~

But it looks the same in the live branch so I don't think we actually need/want this change:

If the next character is not a period, match any word character or one of the following characters: -!#$%&'*+/=?^`{}|~

Reply:
@gewarren, I'm okay with both options (escaping or keeping as it is).

but in my opinion, escaping them is better to see the file correctly in GitHub editor.

Compare both cases:

![image](https://user-images.githubusercontent.com/31348972/67790166-f92f6680-fa7d-11e9-99e6-8675f3add368.png)

![image](https://user-images.githubusercontent.com/31348972/67790184-02203800-fa7e-11e9-807a-81679336325d.png)